frontend/src/react/Router/Router.js

```diff
@@ -1,15 +1,52 @@
 import React from 'react'
 import {Route, Switch as RouterSwitch} from 'react-router';
-import Instructions from "../Instructions/Instructions";
-import CreateExam from "../Exam/CreateExam";
+
+import Instructions from '../Instructions/Instructions';
+import CreateExam from '../Exam/CreateExam';
+import DisplayExam from '../Exam/DisplayExam';
+import LayoutApp from '../Layout/LayoutApp';
+
+// TODO: move this guy in his own file
+const privateRoutes = {
```

Comment:
Great work ! Can be used to pass props to routes !

Reply:
So for example we can pass an Exam to `DisplayExam` ? How do we achieve that ?

Reply:
I don't think we should pass an exam to DisplayExam. We have the id in the route, so we call Redux on the didMount hook in the DisplayExam component. 

If you wanted to pass straight from the Router to DisplayExam, simply add a property to your privateRoutes object, and voila!

What would be the goal of doing that @NansD?

Reply:
yeah I just wanted to have an example for passing props to routes, as @ValentinChion suggested. Didn't aim to talk about a "real" usecase